src/components/DateRangeForm/DateRangeForm.jsx

```diff
@@ -1,9 +1,7 @@
 import React from 'react';
+import PropTypes from 'prop-types';
 
-import Form from 'antd/lib/form';
-import Button from 'antd/lib/button';
-import DatePicker from 'antd/lib/date-picker';
-
+import { Form, Button, DatePicker } from 'antd';
```

Comment:
CircleCI doesn't like this way of importing.. that's why I had to do it the gross way and import them one by one